changelogutils/validator_test.go

```diff
@@ -665,7 +665,56 @@ var _ = Describe("changelog validator utils", func() {
 				Entry("stable release after rc for 1.1", "v1.1.0-rc2", "v1.1.0", validStableReleaseChangelog, noValidationSettingsExist),
 				Entry("stable release after rc for 1.1", "v1.1.0-rc2", "v1.1.0", validStableReleaseChangelog, relaxedValidationSettingsExists),
 				Entry("initial rc after for 1.1", "v1.0.0", "v1.1.0-rc1", validNewFeatureChangelog, noValidationSettingsExist),
-				Entry("initial rc after for 1.1", "v1.0.0", "v1.1.0-rc1", validNewFeatureChangelog, relaxedValidationSettingsExists))
+				Entry("initial rc after for 1.1", "v1.0.0", "v1.1.0-rc1", validNewFeatureChangelog, relaxedValidationSettingsExists),
+				Entry("beta5 before rc1", "v1.0.0-beta5", "v1.1.0-rc1", validNewFeatureChangelog, noValidationSettingsExist),
+				Entry("beta5 before rc1", "v1.0.0-beta5", "v1.1.0-rc1", validNewFeatureChangelog, relaxedValidationSettingsExists),
+				Entry("beta1 before rc1", "v1.0.0-beta1", "v1.1.0-rc1", validNewFeatureChangelog, noValidationSettingsExist),
+				Entry("beta1 before rc1", "v1.0.0-beta1", "v1.1.0-rc1", validNewFeatureChangelog, relaxedValidationSettingsExists))
```

Comment:
i thought we determined that it would be impossible to compare versions with different labels